10x/count/count.wdl

```diff
@@ -78,16 +77,20 @@ task chunk_reads_split {
     /martian/adapters/python/martian_shell.py $stage_path $stage_phase . . run_file_whatever
     
     # Break the chunks into separate files so we can scatter over them
-    jq -c '.chunks[]' _stage_defs | split -l 1 - chunk_
+    jq -c '.chunks[]' _stage_defs | split -l 1 - partition_
```

Comment:
I think we should use this naming for all of the split tasks. The choice of "chunk" in the first place was a little unfortunate because it's used in multiple other ways in the workflow.

Reply:
Fixed, good idea.